src/app/Providers/AppServiceProvider.php

```diff
@@ -8,4 +8,11 @@
 
 class AppServiceProvider extends ServiceProvider
 {
+    public function register(): void
+    {
+    }
+
+    public function boot(): void
+    {
+    }
```

Comment:
Do we need these?

Reply:
Depends on how much boilerplate we want.